tillerinobot/src/main/java/tillerino/tillerinobot/lang/Svenska.java

```diff
@@ -182,23 +183,41 @@ public String setFormat() {
 		return "Syntaxen för att ställa en parameter är !set. Testa !help om du behöver mer hjälp.";
 	}
 	
+	StringShuffler doSomething = new StringShuffler(rnd);
+	
 	@Override
 	public String apiTimeoutException() {
-		return new Default().apiTimeoutException();
+		final String message = "osu! servrarna är jätte sakta just nu, så jag kan inte göra något för dig just nu.";
```

Comment:
add a space as last character of this string (look at default.java to see what I mean)